components/MerchGeneral/merch-general.tsx

```diff
@@ -1,60 +1,83 @@
 import type { FC } from 'react';
-
 import Link from 'next/link';
 import { MerchGeneralT } from 'types';
-import Image from 'next/image';
 import { urlForImage } from '@/lib/sanity';
+import Image from 'next/image';
 
 type MerchGeneralProps = {
   merch: MerchGeneralT;
 };
 
 export const MerchGeneral: FC<MerchGeneralProps> = ({
   merch: { description, link, title, titleLink, imagesList }
```

Comment:
I am confused with naming of imagesList here, and next you use this naming for "getting" imagesList. I propose to change it by 'imageLists' or simply by 'images'. Then you can use 'getImages' for getting them.